dev_out_of_stock/dev_out_of_stock.py

```diff
@@ -0,0 +1,70 @@
+"""Python script that will get a list of products that are out of stock in the dev site."""
+
+from woocommerce import API 
+import os
+import csv
+
+
+
+woo_api = API(
+    url=os.environ.get('url'),
+    consumer_key=os.environ.get('consumer_key'),
+    consumer_secret=os.environ.get('consumer_secret'),
+    wp_api=os.environ.get('wp_api'),
+    version=os.environ.get('version')
+
+
+
+)
+
+
+per_page = 100
+current_page = 1
+count = 0
+skipped_items =[]
+out_of_stock = []
+in_stock = []
+while True:
+        payload = {
+            "per_page": per_page,
+            "page": current_page, #Note: this show the specific page you want to see now
+        }
+
+        all_products = woo_api.get('products', params=payload).json()
+        if not all_products:
+                break 
+        for product in all_products:
+            # #note: This goes through all the products and and ignore all the "collections" products/not simple in product type.
```

Comment:
If you pan to show this to hiring people I say remove the comments and keep it clean.
But also add some print statements. now than then print a message showing what is going on. Like saying `print(f"Getting page number: {current_page}")